theme/islandora-videojs.tpl.php

```diff
@@ -1,15 +1,23 @@
-<video id="islandora_videojs" class="video-js vjs-default-skin" controls
- preload="auto" width="640" height="264" 
+<video id="islandora_videojs" <?php if (empty($responsive)): ?>width="640" height="360"<?php endif; ?>
```

Comment:
Just wondering, it seems that the default height has changed from `264` to `360`. Was that planned?

Reply:
Hi Jared, it was an intentional change from me, I always found the default height to be rather odd since there are hardly any videos with such a wide aspect ratio. But perhaps there are good reasons to stick to 264 that I'm not aware of. Alternatively as I mentioned in the pull request, the height could be left out completely, in which case the player adapts its height automatically.

Reply:
It definitely is a weird size, I was more wondering if this was a keying error. I don't really care either way. I feel like being able to customize the player size using a CSS override should make the responsive mode the better choice all around.

Reply:
@whikloj so you mean removing width and height there and moving to CSS? I feel 640px X 350px is probably the most used one for modern video, but i'm also aware since these are repository needs, old TV will be always in 4:3 or 3:2.. so not sure.

Reply:
No, I mean I (personally) would probably turn on the new option because then I could theme the player with CSS.

But for backwards compatibility, we should leave the default alone.

Though if you wanted to you could create another CSS with the old height and width settings and choose which CSS you add based on the `$responsive` variable. That should be okay too.

Reply:
@whikloj i'm pretty sure the existing default is wrong for any type of video/film, so at least i'm less concerned about backward compat. Even cinemascope with a 21:9 ratio leads to 274 in height. And yes, i personally would also use the new option always.  let me do a blame and ask the original coder that decided those values where the right ones

Reply:
@nhart hi! I did a quick blame on the discussed TPL file and found out that the original dimensions set for the video tag, that are in discussion here, [were set by you](https://github.com/Islandora/islandora_videojs/blame/36cc42219be479f93ff4e0373b875a9cd856e68e/theme/islandora-videojs.tpl.php). Do you remember the rationale on that time and would you be ok if we change that to current standard ratio like 16:9? (640x360?)

Reply:
@DiegoPino  based on the other info in that tpl (eg: my_video_1), I suspect I copied it from an example in video.js

Reply:
And I agree with @whikloj , the default should be left alone.

Reply:
@nhart thanks!. So all good on changing to 16:9 - 640px x 360xp? as default size?

Reply:
hum. I thought @whikloj was referring to leaving what is currently in the tpl there.

Reply:
I was saying leave the default as the (admittedly weird) 264...I don't think it is right, but I hate to alter it out from under anyone.

Reply:
Sorry @whikloj @nhart can't agree on leaving that if we have the chance to fix it here. Really, that size is no standard at all and if there is no rationally/pressing reason other than preserving this (not saying error, but still weird) because of backward compat of not sure what, then well, better to discuss during the next committers call? Its up to @pautri of course.

Reply:
should we ask the community? I wonder whether anyone is using it as is. Indeed video.js seems to use it in their examples, but I don't see the rationale. Changing it to a more common value would mean that fewer people need to override it.

Reply:
I'm using it as it is, and it looks horrible. I would love to change it. I'd say take it out altogether and let the height be responsive! I don't know why we're forcing an aspect ratio at all - that sounds like something that should be done in theme if required. 

I'm sorry that we got caught up with bikeshedding. My suggestion to move things forward wrt this PR is to remove the change in height, so we can merge this (awesome) PR now (#pragmatism).  If there's desire to have a debate about what is an obviously flawed default, which @pautri had the good sense to suggest to fix, let's do that separately.

Reply:
@rosiel @whikloj @nhart instead of asking  @pautri here for more changes on this, let's bring this to tomorrows committers call. I don't want to go back and forth and I feel, as @pautri says that general, community, consensus is better (and that is what our calls are for)

Reply:
...unless @pautri wants to go just width. But: are we sure without a given height, all browsers will expand the container to its given content?

Reply:
@DiegoPino indeed I don't know how well this works in all browsers and what it depends on, I basically found out by chance that it behaves like that. This is also a relatively new feature of the player, I think, as far as I remember it used to be necessary to specify width and height. So the safer option would be to go with a fixed height in non-responsive mode.

Reply:
I'm fine with merging this and tackling other issues after.

However I still have two comments waiting. [Right below for code above](https://github.com/Islandora/islandora_videojs/pull/45/files/df9c272812c3ef71c068382b86fda674eb51b2e5#r150880060) and [further down](https://github.com/Islandora/islandora_videojs/pull/45/files/df9c272812c3ef71c068382b86fda674eb51b2e5#r150880458)

Also I noticed that videojs actually has an [aspectRatio](http://docs.videojs.com/tutorial-options.html#aspectratio) parameter, that you can pass to the viewer to handle sizing. That might be a good follow on to allow `4:3` or `16:9` or whatever.

Reply:
Hi Jared, I fixed those two issues yesterday. The aspectRatio parameter will put the player in "fluid" (responsive) mode. It may well be that the fluid mode also works well with fixed width or non-responsive themes, but to figure that out would require more time than I have at the moment.

Reply:
Sorry @pautri I must have been looking at the wrong tab or something. You have addressed my concerns. When you use videojs without an aspect ratio you can get a stretched video. The one I tested with was a portrait orientation from my phone and because it is narrower than it is tall, it was able to expand and became blurry. This would have to be handled by the CSS.

What I would love to see is this.
https://coolestguidesontheplanet.com/videodrome/videojs/

Reply:
@whikloj no worries. That page you mention is where I found the solution. The data-setup='{"fluid": true}' is an alternative for the aspect ratio classes. For normal video, there's the max-width in the css, but for portrait video one could perhaps add a max-height as wel...